.github/workflows/ci.yml

```diff
@@ -47,3 +47,18 @@ jobs:
 
       - name: Check code with flake8
         run: flake8 .
+
+  docs:
+    name: builds the documentation using Sphinx
```

Comment:
I think you can delete this row. The default `docs` is easier to read in my opinion.